homeassistant/components/ozw/const.py

```diff
@@ -35,6 +35,7 @@
 ATTR_INSTANCE_ID = "instance_id"
```

Comment:
We already use this key for the ozw instance id in the services. We should probably use the same key everywhere.